test/issues.jl

```diff
@@ -0,0 +1,18 @@
+using GLMakie
+using TernaryDiagrams
```

Comment:
I would put the `using` statements only once?

Reply:
Thanks. Corrected.